.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly remember to add node_modules to gitignore which will be generated while setting up linters in your project 😄.